Content.Client/GameObjects/Components/Power/AME/AMEWindow.cs

```diff
@@ -0,0 +1,164 @@
+
+            Title = "Antimatter Control Unit";
+            Contents.AddChild(new VBoxContainer
+            {
+                Children =
+                    {
+                        new HBoxContainer
+                        {
+                            Children =
+                            {
+                                new Label {Text = _localizationManager.GetString("Engine Status") + ": "},
+                                (InjectionStatus = new Label {Text = "Not Injecting"})
+                            }
+                        },
+                        new HBoxContainer
+                        {
+                            Children =
+                            {
+                                (ToggleInjection = new Button {Text = "Toggle Injection", StyleClasses = {StyleBase.ButtonOpenBoth}, Disabled = true}),
+                            }
+                        },
+                        new HBoxContainer
+                        {
+                            Children =
+                            {
+                                new Label {Text = _localizationManager.GetString("Fuel Status") + ": "},
+                                (FuelAmount = new Label {Text = "No fuel inserted"})
+                            }
+                        },
+                        new HBoxContainer
+                        {
+                            Children =
+                            {
+                                (EjectButton = new Button {Text = "Eject", StyleClasses = {StyleBase.ButtonOpenBoth}, Disabled = true}),
+                            }
+                        },
+                        new HBoxContainer
+                        {
+                            Children =
+                            {
+                                new Label {Text = _localizationManager.GetString("Injection amount") + ": "},
+                                (InjectionAmount = new Label {Text = "0"})
+                            }
+                        },
+                        new HBoxContainer
+                        {
+                            Children =
+                            {
+                                (IncreaseFuelButton = new Button {Text = "Increase", StyleClasses = {StyleBase.ButtonOpenRight}}),
+                                (DecreaseFuelButton = new Button {Text = "Decrease", StyleClasses = {StyleBase.ButtonOpenLeft}}),
+                            }
+                        },
+                        new HBoxContainer
+                        {
+                            Children =
+                            {
+                                (RefreshPartsButton = new Button {Text = "Refresh Parts", StyleClasses = {StyleBase.ButtonOpenBoth }, Disabled = true }),
+                                 new Label { Text = _localizationManager.GetString("Core count") + ": "},
+                                (CoreCount = new Label { Text = "0"}),
```

Comment:
All of these text fields need to be localized by wrapping the string in `Loc.GetString()`